bin/run.js

```diff
@@ -27,6 +27,12 @@ try {
 }
 
 // process runfile.js
-console.log('Processing runfile...')
-var runfile = require(path.resolve('./runfile'));
+try {
+    console.log('Processing runfile...')
+    var runfile = require(path.resolve('./runfile'));
```

Comment:
But, how do we differentiate between a "require module" error due to requiring runfile.js and a "require module" error due to requiring another module say x, inside runfile.js ; Like this : 

![runjs](https://cloud.githubusercontent.com/assets/12808424/19516496/e589ff4e-9619-11e6-841f-71c5688ae923.png)

It's because the error type is the same for both! Which is just "Error". So, I guess we need to do a RegExp check on the Error message to see if the require error is due to runfile.js requiring.

Your comments ?